### PR TITLE
Dutch suggestion

### PR DESCRIPTION
**Pull request by Piet2001**
I have made a translation of the dutch docs. 
It's not all, but everything i have now.
If i have time an other time, i will try to also do the other files.

### CODE REVIEW

**Comment by Piet2001** [comment]
I also include a an change in the englisch text.

File: docs/nl/README.md [hunk 1]
```diff
@@ -7,11 +7,38 @@ sidebarDepth: 2
...
+Browser|Link
+-------|----
+Chrome | [Download](https://chrome.google.com/webstore/detail/dhdgffkkebhmkfjojejmpbldmpobfkfo)
+Firefox| [Download](https://addons.mozilla.org/en-US/firefox/addon/tampermonkey/)
+Safari | [Download](https://safari.tampermonkey.net/tampermonkey.safariextz)
+Opera  | [Download](https://addons.opera.com/en/extensions/details/tampermonkey-beta/)
+
+Voor andere browsers u kan Tampermonky downloaden van [tampermonkey.net](https://www.tampermonkey.net/)
+
+::: Waarschuwing
```
Comment (jxn-30) [inline docs/nl/README.md]: Please replace `Waarschuwing` with `warning` so that it is shown as a yellow box.

File: docs/nl/README.md [hunk 2]
```diff
@@ -7,11 +7,38 @@ sidebarDepth: 2
...
+:::
 
 ### Tweede stap: Userscript
+Wanneer u Tampermonkey succesvol in uw browser heeft geïnstalleerd, kunt u [hier](https://github.com/LSS-Manager/lss-manager-v3/raw/master/lssm-v3.user.js) klikken of een nieuw Userscript maken met de volgende inhoud:
+
+<<< lssm-v3.user.js
+
+### Derde stap: Activeren
+
+Hierna volgt de indicator voor een actieve LSS-Manager:! [LSSM-Indikator] (/ img / lssm_navbar.png) Als je in meldkamerspel bent maar deze indicator niet in de rechterbovenhoek ziet, klik dan op het Tampermonkey-pictogram in je browser en controleer of de schakelaar voor LSS-Manager Script op "aan" staat.
```
Comment (jxn-30) [inline docs/nl/README.md]: `! [LSSM-Indikator] (/ img / lssm_navbar.png)` must be`LSSM-Indikator`

**Review by jxn-30 — CHANGES_REQUESTED** [review]
After the merge I will update the NL-Sidebar and Navbars!

File: docs/nl/README.md (hunk 1 above)
Comment (Piet2001) [reply docs/nl/README.md]: https://github.com/LSS-Manager/docs/pull/12/commits/ad634de1c7b25525d5defefffc0eac5f51a3b734

File: docs/nl/README.md (hunk 2 above)
Comment (Piet2001) [reply docs/nl/README.md]: https://github.com/LSS-Manager/docs/pull/12/commits/ad634de1c7b25525d5defefffc0eac5f51a3b734

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**